src/main/java/frc/robot/RobotContainer.java

```diff
@@ -18,8 +20,7 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
+  private final Intake intake = new Intake();
 
   // Replace with CommandPS4Controller or CommandJoystick if needed
   private final CommandXboxController m_driverController =
```

Comment:
Noticed this because it's used down on line 45, even though the controller is technically out of the scope of this PR - this should probably be named `driverController`, rather than `m_driverController`

Reply:
I see! I wasn't quite sure what kind of purpose the "m" served